src/components/activity/screens/ActivityAddEdit/views/ActivityAddEditView.js

```diff
@@ -26,79 +74,56 @@ export default class ActivityAddView extends React.Component {
     ));
   }
```

Comment:
Useless function right ? Event if we do not want to remove it, it should be better to comment it